storage/store.go

```diff
@@ -371,8 +371,8 @@ func NewStore(ctx StoreContext, eng engine.Engine, nodeDesc *proto.NodeDescripto
 	s._splitQueue = newSplitQueue(s.db, s.ctx.Gossip)
 	s.verifyQueue = newVerifyQueue(s.RangeCount)
 	s.replicateQueue = newReplicateQueue(s.ctx.Gossip, s.allocator(), s.ctx.Clock)
-	s.rangeGCQueue = newRangeGCQueue(s.db)
-	s.scanner.AddQueues(s.gcQueue, s.splitQueue(), s.verifyQueue, s.replicateQueue, s.rangeGCQueue)
+	s._rangeGCQueue = newRangeGCQueue(s.db)
+	s.scanner.AddQueues(s.gcQueue, s._splitQueue, s.verifyQueue, s.replicateQueue, s._rangeGCQueue)
```

Comment:
Just because we can't have an accessor with same name as a member variable. I inherited this style.